debian/changelog

```diff
@@ -1,3 +1,215 @@
+pulseaudio (1:6.0-1endless1) eos; urgency=low
```

Comment:
Oh, and just for background, the reason to maintain the endlessX revision before was to keep the upgrade path appropriate when switching from the pure OBS debian package to the jenkins git flow.